master-firmware/src/lwip_bindings/lwipthread.c

```diff
@@ -69,19 +69,40 @@ limitations under the License.
 #include <lwip/stats.h>
 #include <lwip/snmp.h>
 #include <lwip/tcpip.h>
+#include <parameter/parameter.h>
 #include "netif/etharp.h"
 #include "netif/ppp/pppoe.h"
 #include "can/can_uwb_ip_netif.hpp"
+#include "main.h"
+#include "uid.h"
+#include "crc/crc16.h"
```

Comment:
[nitpicking] `#include <crc/crc16.h>` and move above since it's from the `lib` folder